src/components/calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="calculator-input">
+        <div className="result"><span>0</span></div>
+      </div>
+      <div className="calculator-keypad">
+        <div className="row">
+          <button type="button" className="keys-function">AC</button>
+          <button type="button" className="keys-function">+/-</button>
+          <button type="button" className="keys-function">%</button>
+          <button type="button" className="keys-operators">+</button>
+        </div>
+        <div className="row">
+          <button type="button" className="keys-numbers">9</button>
+          <button type="button" className="keys-numbers">8</button>
+          <button type="button" className="keys-numbers">7</button>
+          <button type="button" className="keys-operators">x</button>
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer your `Calculator.js` should have at least 1 child component. Please make one child component and use it on the `Calculator.js`.